db/models.py

```diff
@@ -50,3 +55,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="users")
```

Comment:
The `related_name` for the `user` field in the `Order` model is incorrectly set to `users`. This should be something like `orders` to reflect the relationship correctly and avoid conflicts with the `User` model.